apps/sync/test/unit/sync-engine/syncengine_test.js

```diff
@@ -155,7 +155,10 @@ ld be a Function`);
     test('Passes options to the DataAdapter', function(done) {
       var se = new SyncEngine(SynctoServerFixture.syncEngineOptions);
       se.syncNow({ history: { readonly: true } }).then(function() {
-        expect(AdapterMock.options).to.deep.equal({ readonly: true });
+        expect(AdapterMock.options).to.deep.equal({
+          readonly: true,
+          userid: SynctoServerFixture.xClientState
+        });
```

Comment:
We now pass the `userid` to the DataAdapters so that they can use that as a prefix for asyncStorage records.